### DISCUSSION TITLE
Weird suggestion

### DISCUSSION THREAD

**Discussion by Bradley-r-D** [community/community#7237] [Copilot Conversations]
I have the following code:
![VS Code Screenshot](https://user-images.githubusercontent.com/43014079/140233811-212b6909-ae7c-4868-99f4-87db6852c9b0.png)

Why did it suggest the same code when I had two different comments?

**Comment by kaydioactive** [comment]
Could be ML -- learning takes time, I think 12 months from now it wont suggest same code.
